apps/studio/components/interfaces/Realtime/Inspector/useRealtimeMessages.ts

```diff
@@ -190,7 +203,7 @@ export const useRealtimeMessages = (
 
         newChannel.unsubscribe()
         setChannel(undefined)
-        setRealtimeConfig({ ...config, enabled: false })
+        setRealtimeConfig({ ...config, channelName: '', enabled: false })
```

Comment:
just a nit here - any reason why we need to reset the channel name input field if there's an error?
just thinking from the user's POV - if say this turned out to be a mistake and i wanna test, i'd need to key in the channel name again then